uber/site_sections/registration.py

```diff
@@ -165,15 +164,9 @@ def qrcode_generator(self, data):
         this function.  Or, offload image generation to a dedicated microservice that replicates this functionality.
 
         """
-        if not c.QR_CODE_PASSWORD or not qr_cipher:
-            # Don't generate a 2D barcode if we aren't configured for encryption
-            return '2D barcode password not set, cannot generate images'
-
-        encrypted_data = c.EVENT_QR_ID + bytes.decode(binascii.hexlify(qr_cipher.encrypt(uuid.UUID(data).hex)))
-
         checkin_barcode = treepoem.generate_barcode(
             barcode_type='azteccode',
-            data=encrypted_data,
+            data=c.EVENT_QR_ID + str(data),
```

Comment:
> Unfortunately Python's uuid.uuid4() is NOT cryptographically random all the time.

In practice it pretty much always is.  When you refer to "low entropy" I assume you're referring to the outdated myth that ``/dev/urandom`` runs out of entropy on modern systems, which is not the case.
https://sockpuppet.org/blog/2014/02/25/safely-generate-random-numbers/

> The Birthday Paradox makes collision significantly more likely than it would appear

Agreed, but I stand by my statement about never ever seeing a collision and not needing to worry about it.